lib/utils/Database.cpp

```diff
@@ -35,6 +35,7 @@ bool
 Database::create(const std::string& file_name)
 {
     std::cout << "Creating file: " << file_name << std::endl;
+    return true;
```

Comment:
The base class doesn't do anything. Wouldn't it make more sense to make this a pure virtual function?

Reply:
Also, it looks like `fileExists` is unused and can be removed.